common/beerocks/bcl/source/network/socket.cpp

```diff
@@ -597,39 +597,29 @@ size_t Socket::getBytesWritePending()
 }
 #endif
 
-SocketPoll::SocketPoll(uint timeout) : m_epoll_fd(-1), m_timeout(timeout)
+SocketPoll::SocketPoll(uint timeout) : m_timeout(timeout)
```

Comment:
```suggestion
SocketPoll::SocketPoll(uint32_t timeout) : m_timeout(timeout)
```

Reply:
Was already fixed in a previous fixup commit